eap-latest-setup.md

```diff
@@ -1697,9 +1697,79 @@ directory with:
 
     mkdir -p /mnt/registry
 
+### Creating user account for registry
+Each pod runs under particular service account. Three such accounts are created for you
+by default in each project:
+
+    os get serviceaccounts
+    NAME       SECRETS
+    builder    2
+    default    2
+    deployer   2
+
+`builder` service account is required by builder pods and `deployer` by
+deployment pods. The `default` is used for all the other pods unless
+overridden.
+
+Each service account can be given security context constraints (SCC). They
+allow to specify SELinux context for pods, limit users used to run them etc.
+To list them, type:
+
+    oc get scc
+    NAME         PRIV      CAPS      HOSTDIR   SELINUX     RUNASUSER
+    privileged   true      []        true      RunAsAny    RunAsAny
+    restricted   false     []        false     MustRunAs   MustRunAsRange
+
+By default, newly created pods are *restricted* which protects users from compromising
+the system. However, restricted pod does not allow you to mount host directory. The
+registry pod needs be run under *privileged* service account. Since all currently
+existing have already their use, let's set up a new one as `root`:
+
+    oc create -f - <<EOF
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: registry
+    EOF
+
+Next step is to make it *privileged*. This is achieved by adding service
+account's user name to a list of users of *privileged* SCC. Such a name looks
+like this:
+
+    system:serviceaccount:<namespace>:<name>
+
+Where `<namespace>` is `default` because we set up registry in the *default*
+namespace and `<name>` is whatever we used in previous `oc create` command.
+
+Type following command and add the user name:
+
+    oc edit scc priveliged
```

Comment:
> is privileged necessary just to get host dir? I'd rather have an scc that allows host dir specifically and add the registry service account to that

The ideal would be to create an SCC with the minimal set of permissions to allow host dir and leave the privileged SCC limited to cluster administrators only.  Ultimately, we do not recommend this way of running the registry though so I wonder about the value of documenting it so thoroughly in the context of the registry and not just in a generic "here's how you add a service account to another SCC".  We could provide detailed documentation about persistent storage as the correct way to run the registry.

All that said, in the admin/router command it also requests a privileged container if you're using a host mount (https://github.com/pweil-/origin/blob/master/pkg/cmd/admin/registry/registry.go#L250-250).  @ncdc - I imagine there was a reason for this?  Dir permissions perhaps?  I'd be interested to see what happens if we removed that and tested by enabling host dir in the restricted SCC.